stac_fastapi/pgstac/setup.py

```diff
@@ -16,6 +16,7 @@
     "asyncpg",
     "buildpg",
     "brotli_asgi",
+    "pygeofilter @ git+https://github.com/geopython/pygeofilter@main#egg=pygeofilter",
```

Comment:
Do we have a timeline for release?
If not, we should at least pin this to a commit.

Reply:
I don't know timeline for release, but these changes are in the tagged version v0.1.1